driver-cql-shaded/src/test/java/io/nosqlbench/generators/cql/lang/ParserForCqlTest.java

```diff
@@ -37,15 +38,14 @@ public void parseAll() {
                 URL url = resources.nextElement();
                 System.out.println("url=" + url.toExternalForm());
                 Path path = Paths.get(url.toURI());
-                Files.walk(path, FileVisitOption.FOLLOW_LINKS)
-                    .filter(p -> !Files.isDirectory(p, LinkOption.NOFOLLOW_LINKS))
-                    .forEach(subpaths::add);
+                try (Stream<Path> fileStream = Files.walk(path, FileVisitOption.FOLLOW_LINKS)) {
+                    fileStream.filter(p -> !Files.isDirectory(p, LinkOption.NOFOLLOW_LINKS))
+                            .forEach(subpaths::add);
+                }
             }
```

Comment:
I'm not super clear on the benefit of this additional try scope.

Reply:
```
The returned stream encapsulates one or more DirectoryStreams. If timely disposal of file system resources is required, the try-with-resources construct should be used to ensure that the stream's close method is invoked after the stream operations are completed.
```
https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#walk-java.nio.file.Path-java.nio.file.FileVisitOption...-

so making sure resources are cleared up immediately

Reply:
Yes, somehow I missed the twist on try-with-resources. I see it now.